material-overrides/assets/stylesheets/kluster.css

```diff
@@ -1,68 +1,108 @@
 :root {
-  --md-border-radius: 0.1rem;
-  --md-border-width: 0.1rem;
+  /* General styles */
+  --hero-border-radius: 2rem;
+  --hero-border-width: 1px;
+
+  --md-border-radius: 1.3em;
+  --md-border-width: 1px;
+
+  /* Buttons styles NEED TO APPLY */
+  --feat-button-radius: 0.63em;
+  --feat-button-height: 45px;
+  --feat-button-padding: 0.4rem 1.2rem;
+  --feat-button-width: 1px;
 }
 
 :root,
-[data-md-color-scheme='default'] {
+[data-md-color-scheme="default"] {
+  /*New Color Palet*/
+  --dark-green: #1a2428;
+  --light-green: #83a2af;
+  --light-green-transparent-33: rgb(58 74 81 / 33%);
+  --light-green-30: #83a2af4d;
+  --dark: #060809;
+  --electric-indigo: #6525f8;
+  --mint: #5feda9; /* only for badge external */
+  --blue-site: #6e8dff; /* blue from the kluster.ia site */
+  --blue: #0a40ff; /* New blue color from the figma file */
+  --blue-framer: #0c40ff; /* Framer blue color */
+  --ligth-transparent-10: rgba(
+    255,
+    255,
+    255,
+    0.1
+  ); /* use for border-top on nav-links */
   --light: #ffffff;
+  /*old color palet*/
   --off-white: #f6f6f6;
   --platinum: #d6d8d9;
```

Comment:
Using this for a border color. Let's remove this and add a border color variable and use that for all the borders

Reply:
Add variables for: border-color, border-width, and border (i.e., var(--border-width) solid var(--border-color))